x-pack/platform/packages/shared/kbn-streamlang/types/processors/index.ts

```diff
@@ -274,11 +274,14 @@ export const removeProcessorSchema = processorBaseWithWhereSchema.extend({
 
 export interface DropDocumentProcessor extends ProcessorBaseWithWhere {
   action: 'drop_document';
+  // TODO - double check processor works with this field similarly to the other processors that have it
+  ignore_missing?: boolean;
```

Comment:
`ignore_missing` defines the behavior when **Source Field** is missing. Since, `drop_document` processor doesn't have a source field, it shouldn't be there on UI. Also the [ingest processor](https://www.elastic.co/docs/reference/enrich-processor/drop-processor) doesn't have the `ignore_missing` param.

Or is there more to it I am missing?